<?xml version='1.0'?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget name="MainDialog" class="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cient Application</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getfile_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Get File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="status_Text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
